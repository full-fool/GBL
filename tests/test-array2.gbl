class mymain extends Main{
int main()
{
  string stringArray[5];
  stringArray[1]="hello";
  stringArray[2]="world";
  int i;
  for(i = 0; i < 5; i = i + 1){
    printlns(stringArray[i]);
  }
  return 0;
}
}