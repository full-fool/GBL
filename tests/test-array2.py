import sys
class mymain:
    def __init__(self):
        pass

    def main(self):
        stringArray = [ None ] * 5
        stringArray[1]="hello"
        stringArray[2]="world"
        i = None
        i = 0
        while ((i) < (5)):
            print (stringArray[i])
            i = (i) + (1)
        return 0
        pass


if __name__ == "__main__":
    _main = mymain()
    _main.main()

