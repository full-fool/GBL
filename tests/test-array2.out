None
hello
world
None
None
